src/loggers.jl

```diff
@@ -150,7 +150,10 @@ that prints to stdout.
 # Returns
 * `Logger`: the root logger.
 """
-config!(level::AbstractString; kwargs...) = config!(Logger("root"), level; kwargs...)
+function config!(level::AbstractString; kwargs...)
+    DEFAULT_LOG_LEVEL = level
```

Comment:
This might need a `global DEFAULT_LOG_LEVEL` for 0.7, can't remember offhand, but it doesn't hurt either. :slightly_smiling_face:

Reply:
1) Alex is right
2) You don't need to deal with the references to `_loggers` as they mutate instead of reassigning

Reply:
In retrospect, I should probably just not allow modification of the `DEFAULT_LOG_LEVEL` as:

1) This causes a segfault at [loggers.jl#L32](https://github.com/invenia/Memento.jl/blob/c9305dec6b3ba8b6bcc694144bb3622b4f4867ba/src/loggers.jl#L32)
2) The condition that line is checking for can no longer be guaranteed.

Alternatively, we could probably use a `Ref` and remove that check, but that might be better as a separate PR.

Reply:
I was actually going to suggest using a `Ref` for `DEFAULT_LOG_LEVEL`. In general it seems to be The Best Way to deal with mutable globals. But I agree a separate PR makes sense for that.